sdk/core/azure-core/src/main/java/com/azure/core/http/HttpHeaderName.java

```diff
@@ -412,4 +412,9 @@ public boolean equals(Object obj) {
      * {@code x-ms-client-request-id}
      */
     public static final HttpHeaderName X_MS_CLIENT_REQUEST_ID = fromString("x-ms-client-request-id");
+
+    /**
+     * {@code x-ms-encryption-context}
+     */
+    public static final HttpHeaderName X_MS_ENCRYPTION_CONTEXT = fromString("x-ms-encryption-context");
```

Comment:
This is not the correct place for this header name, we should not be editing the Core package.